src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -53,6 +53,7 @@ enum {
 	XOCL_DSAFLAG_VERSAL			= (1 << 11),
 	XOCL_DSAFLAG_MPSOC			= (1 << 12),
 	XOCL_DSAFLAG_NOSC			= (1 << 13),
+	XOCL_DSAFLAG_OT_OVERRIDE		= (1 << 14),
```

Comment:
We can still use "XOCL_DSAFLAG_NOSC" here since this change is only applicable for SmartSSD U.2 board for now.
But, if other platforms do require this kind of change in future, so added new flag "XOCL_DSAFLAG_OT_OVERRIDE" for this. Please let me know if XOCL_DSAFLAG_NOSC flag only to be used in the sysmon driver for this change also. I can make necessary change.

Reply:
We should avoid using global flag if possible. Could you remove this flag? Instead, you may define a private data for sysmon subdevice to achieve this. Please search "xocl_flash_privdata" in devices.h as an example.

Reply:
Hi @houlz0507 , I have taken care of your suggestions in the change, can you please review the change again and let me know. Thank you.